utils/hct/gen_intrin_main.txt

```diff
@@ -442,6 +442,14 @@ $classT [[ro]] SampleBias(in sampler s, in float<1> x, in float bias, in int<1>
 $classT [[]] SampleBias(in sampler s, in float<1> x, in float bias, in int<1> o, in float clamp, out uint_only status) : tex1d_t_bias_o_cl_s;
 $classT [[]] SampleGrad(in sampler s, in float<1> x, in $type2 ddx, in $type2 ddy, in int<1> o, in float clamp) : tex1d_t_dd_o_cl;
 $classT [[]] SampleGrad(in sampler s, in float<1> x, in $type2 ddx, in $type2 ddy, in int<1> o, in float clamp, out uint_only status) : tex1d_t_dd_o_cl_s;
+float_like [[]] SampleCmpBias(in sampler_cmp s, in float<1> x, in float compareValue, in float bias) : tex1d_t_comp_bias;
+float_like [[]] SampleCmpBias(in sampler_cmp s, in float<1> x, in float compareValue, in float bias, in int<1> o) : tex1d_t_comp_bias_o;
```

Comment:
It's a nit, but these don't need to be on the bottom just because they are newest. I think it would be clearer if they were added as the SampleCmpLevel variants were before them, just under the two variants that these correspond to. So these two would go on what is now line 428.